types/tailwindcss/index.d.ts

```diff
@@ -0,0 +1,7 @@
+// Type definitions for tailwindcss 2.0
+// Project: https://tailwindcss.com
```

Comment:
Can you please update link to https://github.com/tailwindlabs/tailwindcss

Reply:
I have done so